.github/workflows/_e2e_tests.yaml

```diff
@@ -67,15 +113,33 @@ jobs:
         with:
           password: ${{ secrets.PULP_PASSWORD }}
 
-      - name: run ${{ matrix.test }}
+      # The only reason we need those 2 is the fact that TEST_KONG_CONTROLLER_IMAGE_LOAD
+      # is ultimately passed down to ktf's loadimage.Addon which uses "kind load docker-image ..."
+      # which fails when an image is not available on the cluster.
+      # Hence the non local version (where the image is pulled from an external registry)
+      # doesn't define TEST_KONG_CONTROLLER_IMAGE_LOAD.
+      # todo: consider adapting go code to handle both cases (local and non local) using a single env var.
```

Comment:
Can we create an issue out of this TODO and put the link to it here?

Reply:
Let's solve it with https://github.com/Kong/kubernetes-ingress-controller/pull/3741.

Reply:
👍 Do you mind adding a link in the TODO comment?

Reply:
As we managed to merge https://github.com/Kong/kubernetes-ingress-controller/pull/3741 already, I've just adapted to it here.